cmd/templates-api/decode.go

```diff
@@ -0,0 +1,12 @@
+package main
+
+import (
+	"net/http"
+
+	goaHTTP "goa.design/goa/v3/http"
+)
+
+// decider decodes request.
+func decoder(r *http.Request) goaHTTP.Decoder {
+	return goaHTTP.RequestDecoder(r)
+}
```

Comment:
Potreboval som vlastny `encoder`, tak som spravil aj vlastny `decoder`, ak by sme do buducna potrebovali nieco zmenit, nech je to jednotne.